src/config.rs

```diff
@@ -15,6 +15,22 @@ pub fn default_config_path(mut home_dir: PathBuf) -> String {
     home_dir.to_str().unwrap().to_owned()
 }
 
+#[cfg(not(windows))]
+pub fn global_config_path() -> String {
+    let mut path = PathBuf::from("/etc/puppetlabs/client-tools");
+    path.push("puppetdb");
+    path.set_extension("conf");
+    path.to_str().unwrap().to_owned()
+}
+#[cfg(windows)]
+pub fn global_config_path() -> String {
+    let mut path = PathBuf::from("C:\\ProgramData\\PuppetLabs\\client-tools");
```

Comment:
Just a heads up, it looks like this folder location could be different depending on what version of windows the user is running this on. We were using a function from boost in c++ to determine the PROGRAMDATA folder location (which ended up having a bug) but you might want to look into that: https://tickets.puppetlabs.com/browse/PE-16040